Intersect.Client/Core/Fade.cs

```diff
@@ -64,6 +76,11 @@ public static void Update()
                 {
                     sCurrentAction = FadeType.None;
                     sFadeAmt = 0f;
+
+                    if (InformServer)
```

Comment:
Move the `InformServer` check into `InformServerOfCompletion()`

Reply:
done